doc/user/deploying.md

```diff
@@ -0,0 +1,88 @@
+# Deploying TruffleRuby
+
+If you're attempting to experiment with deploy TruffleRuby to production we'd
+encourage you to contact us so we can help you understand what is possible at
+the moment and to help solve any issues for you.
+
+## TruffleRuby configurations
+
+There are two main configurations of TruffleRuby - *native* and *JVM*. It's
+important to understand the different configurations of TruffleRuby, as each has
+different capabilities and performance characteristics. You should pick the
+execution mode that is appropriate for your application.
+
+When distributed as part of GraalVM, TruffleRuby by default runs in the *native*
+configuration. In this configuration, TruffleRuby is ahead-of-time compiled to a
+standalone native executable. This means that you don't need a JVM installed on
+your system to use it. The advantage of the native configuration is that it
+[starts about as fast as MRI](../contributor/svm.md), it may use less memory,
+and it becomes fast in less time. The disadvantage of the native configuration
```

Comment:
I'd add "than the JVM configuration," as on my first read I thought you were claiming lower memory than MRI.

Reply:
Fixing.